airgun/views/hostgroup.py

```diff
@@ -17,6 +17,18 @@
 from airgun.widgets import RadioGroup
 
 
+class ActivationKeyDropDown(ActionsDropdown):
+    dropdown = Text('.//*[self::div]')
```

Comment:
This makes the dropdown widget locator VERY broad, when the `ActionsDropdown` locator is currently specific for a few xpath attributes with some variation.

Is the activation key page component missing all of those attributes used in the `ActionsDropdown.dropdown` locator? (Below for reference)

```
dropdown = Text(
        ".//*[self::a or self::button][contains(@class, 'dropdown-toggle') or "
        "contains(@ng-click, 'toggleDropdown')][contains(@class, 'btn')]"
        "[*[self::span or self::i][contains(@class, 'caret')]]"
    )
```

Reply:
I will try to see if I can build the more solid locator

Reply:
@mshriver .//*[self::div] looks inside the drop down locator, I did not find anything that rock solid